docs/modelserving/v1beta1/transformer/collocation/README.md

```diff
@@ -13,13 +13,16 @@ KServe by default deploys the Transformer and Predictor as separate services, al
 2. Your cluster's Istio Ingress gateway must be [network accessible](https://istio.io/latest/docs/tasks/traffic-management/ingress/ingress-control/).
 3. You can find the [code samples](https://github.com/kserve/kserve/tree/master/docs/samples/v1beta1/transformer/collocation) on kserve repository.
 
-## Deploy the InferenceService
+## Collocation with custom container
+### Deploy the InferenceService
 
 Since, the predictor and the transformer are in the same pod, they need to listen on different ports to avoid conflict. `Transformer` is configured to listen on port 8080 (REST) and 8081 (GRPC) 
 while, `Predictor` listens on port 8085 (REST). `Transformer` calls `Predictor` on port 8085 via local socket. 
 Deploy the `Inferenceservice` using the below command.
 
-```bash
+Note that, readiness probe is specified in the transformer container. This due to the limitation of Knative. You can provide `--enable_predictor_health_check` argument to allow the transformer container to check the predictor health as well. This will make sure that both the containers are healthy before the isvc is marked as ready.
```

Comment:
```suggestion
Note that, readiness probe is specified in the transformer container. This is due to the limitation of Knative. You can provide `--enable_predictor_health_check` argument to allow the transformer container to check the predictor health as well. This will make sure that both the containers are healthy before the isvc is marked as ready.
```

Reply:
it is not a limitation of Knative, knative adds the default readiness TCP check for the entrypoint container, but ye if you want a proper HTTP check then you need to add it explicitly.